modules/reference/pages/internal-metrics-reference.adoc

```diff
@@ -65,6 +65,46 @@ Can indicate latency caused by disk operations.
 
 ---
 
+=== vectorized_kafka_quotas_balancer_runs
+
+Number of times the throughput quota balancer has executed.
+
+*Type*: counter
+
+---
+
+=== vectorized_kafka_quotas_quota_effective
+
+Current effective quota, in bytes per second.
```

Comment:
what quota @greg-szrama  the node throughput quota?

Reply:
> what quota @greg-szrama the node throughput quota?

This is primarily for the quota balancer, but it's left intact. Without the quota balancer, it's node throughput / shard count. It may be aggregated by shard if `aggregate_metrics` is true.